tests/Nox.Tests/DatabaseIntegrationTests/SqlServerIntegrationTests.cs

```diff
@@ -26,6 +27,7 @@ public void GeneratedEntity_SqlServer_CanSaveAndReadFields_AllTypes()
             MoneyTestField = Money.From(money, currencyCode),
             CountryCode2TestField = CountryCode2.From(countryCode2),
             AreaTestField = Area.FromSquareMeters(areaInSquareMeters),
+            LanguageCodeTestField = LanguageCode.From(languageCode)
```

Comment:
Could you please add comma to end of this line so that new lines coming from other branches can be added automatically without any error?